cves/kernel/CVE-2020-24394.yml

```diff
@@ -139,10 +129,14 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: |
+    The original code was unit tested, but it wasn't involved
```

Comment:
Do you mean that the other parts of the source code were unit tested, but this part of the source code wasn't unit tested? I would just be more clear.

Reply:
Fixed